packages/clerk-js/src/core/resources/Passkey.ts

```diff
@@ -0,0 +1,147 @@
+import type { PasskeyJSON, PasskeyResource, PasskeyVerificationResource } from '@clerk/types';
+
+import { unixEpochToDate } from '../../utils/date';
+import {
+  bufferToBase64Url,
+  handlePublicKeyCreateError,
+  isWebAuthnPlatformAuthenticatorSupported,
+} from '../../utils/passkeys';
+import { BaseResource, ClerkRuntimeError, PasskeyVerification } from './internal';
+
+export class Passkey extends BaseResource implements PasskeyResource {
+  id!: string;
+  pathRoot = '/me/passkeys';
+  credentialId: string | null = null;
+  verification: PasskeyVerificationResource | null = null;
+  name: string | null = null;
+  lastUsedAt: Date | null = null;
+  createdAt!: Date;
+  updatedAt!: Date;
+
+  public constructor(data: PasskeyJSON) {
+    super();
+    this.fromJSON(data);
+  }
+
+  private static async startRegistration() {
+    return await BaseResource._fetch({
+      path: `/me/passkeys`,
```

Comment:
Should we go with `/me/passkeys/prepare_verification`? (and similarly `/me/passkeys/attempt_verification` below?

Reply:
Yes, thanks for reminding me